notebooks/person-tracking-webcam/person-tracking.ipynb

```diff
@@ -172,23 +172,34 @@
     "# A directory where the model will be downloaded.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        download_ir_model(download_det_model_xml_url, Path(base_model_dir) / detection_model_name / precision)
</pre><p>download_ir_model accepts xml url, and then download bin and xml files. It checks separately if each of them already exists and downloads if not. So, it is enough to remove<code>if not detection_model_path.exists():</code> and  <code>if not reidentification_model_path.exists():</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3012/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>